oioswift/common/middleware/autocontainerbase.py

```diff
@@ -202,6 +206,21 @@ def check_container_obj(alternative):
             env_modifier=modify_copy_from,
             alt_checker=check_container_obj)
 
+    def _skip_metadata(self, env):
```

Comment:
This method does not skip anything. It would be better called `_mock_infocache`.